<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-70</x>
     <y>320</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>261</width>
     <height>240</height>
    </rect>
   </property>
   <property name="title">
    <string>Training data</string>
   </property>
   <widget class="QCheckBox" name="save_steps_chk_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Save steps</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="save_folder_edt_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_save_folder_btn_">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Folder to save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="start_index_edt_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Start index</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Finish index</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="finish_index_edt_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>99999</string>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="autorestart_chk_">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto-restart game</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Error tracking</string>
   </property>
   <widget class="QCheckBox" name="unexpected_error_chk_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Save unexpected fault</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="wrong_mine_chk_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Save steps before wrong mine</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="probability_steps_chk_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Save probability steps</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>OnAccept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnAccept()</slot>
 </slots>
</ui>
